lib/components/Field/Field.module.scss

```diff
@@ -76,9 +76,12 @@ $line-height: 5*$grid-size;
 .field-error {
     @include md-box(block, ellipsis);
     height: $form-error-height;
-    width: 100%;
+    width: fit-content;

```

Comment:
why?, if the error is huge, will this make the width bigger than the field?

Reply:
reverted